_posts/for-developers/apps/vocabularies/2019-01-01-01_quickstart-creation.md

```diff
@@ -17,44 +19,72 @@ Let's start by stating in English what we'd like to put in our vocabulary:
+As we can see, identifiers quickly become unpleasant to read when they are IRIs,
+so RDF introduces the notion of prefixes (a simple concept borrowed from XML
+namespaces). From now on we'll use the prefix `obelisk:` to stand in for our
+vocabulary identifier `http://w3id.org/obelisk/`, which means our vocabulary
+now looks like:
+- Use the prefix 'obelisk:' for our vocabulary identifier http://w3id.org/obelisk/.
+- An [obelisk:Obelisk](http://w3id.org/obelisk/Obelisk) is [obelisk:ownedBy](http://w3id.org/obelisk/ownedBy) an [obelisk:Person](http://w3id.org/obelisk/Person).
+- An [obelisk:Obelisk](http://w3id.org/obelisk/Obelisk) is [obelisk:builtBy](http://w3id.org/obelisk/builtBy) an [obelisk:Sculptor](http://w3id.org/obelisk/Sculptor).
+- An [obelisk:Obelisk](http://w3id.org/obelisk/Obelisk) has an [obelisk:height](http://w3id.org/obelisk/height), which is a numerical value.
+
+### Things, and Properties of Things
+
+From the above we can see that we want to describe both 'Things' (e.g.
+Obelisks and Sculptors), and the 'Properties' of those things (e.g. their
+height, or who owns them). RDF allows us to explicitly distinguish between
+these by referring to 'things' as Classes, and 'properties' as, well, 
+Properties!
+
+### Defining Classes of Things
+
+In RDF, the general things that we can talk about are called __Classes__.
+Therefore everything that went into a bubble in our diagram above is a Class, so
+we could add the following to our vocabulary:
 - [obelisk:Obelisk](http://w3id.org/obelisk/Obelisk) is a Class.
 - [obelisk:Person](http://w3id.org/obelisk/Person) is a Class.
 - [obelisk:Sculptor](http://w3id.org/obelisk/Sculptor) is a Class.
 
-If we look at these sentences, they are structured exactly like the ones from the rest of our vocabulary. Let us underline the important bits in the same way:
+If we look at these sentences, they are structured exactly like the ones from
+the rest of our vocabulary. Let us underline the important bits in the same way:
 - [obelisk:Obelisk](http://w3id.org/obelisk/Obelisk) [is a](???) [class](???).
 - [obelisk:Person](http://w3id.org/obelisk/Person) [is a](???) [class](???).
 - [obelisk:Sculptor](http://w3id.org/obelisk/Sculptor) [is a](???) [class](???).
 
-What we need now are IRI for the "[is a](???)" property and the "[Class](???)" Class. Fortunately, these are defined in the RDF and RDFS vocabularies: "[is a](???)" is defined by [rdf:type](http://www.w3.org/1999/02/22-rdf-syntax-ns#type), and "[class](???)" by [rdfs:Class](http://www.w3.org/2000/01/rdf-schema#Class). Therefore, we could write:
+What we need now are IRIs for the "[is a](???)" property and the "[Class](???)"
+Class. Fortunately, these are defined in the RDF and RDFS vocabularies:
+"[is a](???)" is defined by
+[rdf:type](http://www.w3.org/1999/02/22-rdf-syntax-ns#type), and "[class](???)"
```

Comment:
I think it would be better to use Turtle's short `*a*` than `rdf:type`, even though it is a syntax specific thing.

Reply:
I'd prefer not to use any syntactic sugar in this quick intro, in order to limit the quantity of concepts introduced at once. The `a` shorthand is introduced in `2019-01-01-02_quickstart-usage.md`, which describes how to use your vocabulary to describe data.